packages/accordion/src/styles/default.module.css

```diff
@@ -0,0 +1,94 @@
+@import '../../../themes/src/default.css';
+
+.accordion {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
```

Comment:
Not sure if `height: 100%` is needed. This leads to a break when the component is being used in a grid for some reason